src/main/features/claim/index.ts

```diff
@@ -24,11 +25,11 @@ export class Feature {
   enableFor (app: express.Express) {
     app.all('/claim/*', claimIssueRequestHandler())
     app.all(
-      /^\/claim\/(?!start|amount-exceeded|.+\/confirmation|.+\/receipt|.+\/defendant-response|eligibility$|eligibility\/eligible).*$/,
+      /^\/claim\/(?!start|amount-exceeded|.+\/confirmation|.+\/receipt|eligibility$|eligibility\/eligible).*$/,
       DraftMiddleware.requestHandler(new DraftService(), 'claim', 100, (value: any): DraftClaim => {
         return new DraftClaim().deserialize(value)
       }))
-
+    app.all(/^\/claim\/.+\/receipt$/, ClaimMiddleware.retrieveByExternalId)
```

Comment:
I would prefer:
```
app.all('/claim/*/receipt/, ClaimMiddleware.retrieveByExternalId)
```
easier to read
REC

Reply:
Done. Damian recommended to remove the trailing slash after receipt so its now
    app.all('/claim/*/receipt', ClaimMiddleware.retrieveByExternalId)